.github/workflows/build-image.yml

```diff
@@ -30,7 +30,8 @@ jobs:
             latest=auto
           tags: |
             type=edge,branch=main
-            type=match,pattern=v(.*),group=0
+            type=match,pattern=v(\d.\d.\d)(?!rc),group=0
+            type=match,pattern=v(\d.\d.\d)(rc),group=1,value={{ tag }}
```

Comment:
Since the original capture group will no longer capture `v1.2.3rc4`, a second match is added to capture `rc` and set the docker tag value to the release candidate git tag.